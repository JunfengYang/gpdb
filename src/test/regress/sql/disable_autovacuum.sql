-- start_ignore
\! gpconfig -c autovacuum -v off --skipvalidation;
\! gpstop -ari;
-- end_ignore
\! psql postgres -c "show autovacuum;";
