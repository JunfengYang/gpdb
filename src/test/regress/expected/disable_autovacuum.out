-- start_ignore
\! gpconfig -c autovacuum -v off --skipvalidation;
\! gpstop -ari;
-- end_ignore
\! psql postgres -c "show autovacuum;";
 autovacuum 
------------
 off
(1 row)

